### ISSUE TITLE
Merge-Sort

### ISSUE THREAD

**Issue by abirbhattacharya82** [abirbhattacharya82/Hacktoberfest22-DSA-CP-Code-Dump#30]
Sort an Array using Merge Sort in JAVA

**Comment by abirbhattacharya82** [comment]
You are Good to Go 👍🏽

**Comment by flow6979** [comment]
hey, brother can you please assign this to me? 
Thank you.

**Comment by abirbhattacharya82** [comment]
Hi @flow6979 this issue is already assigned. However if you want to work on the same thing but in different language then create an issue I will assign you with the relevant tags there

**Comment by flow6979** [comment]
@abirbhattacharya82 can i create issue on KMP algorithm??

**Comment by abirbhattacharya82** [comment]
@flow6979 Yeah definately. I will assign it to you with the relative labels.
